src/scala/ly/stealth/mesos/kafka/Scheduler.scala

```diff
@@ -125,6 +125,24 @@ object Scheduler extends org.apache.mesos.Scheduler {
 
   def frameworkMessage(driver: SchedulerDriver, executorId: ExecutorID, slaveId: SlaveID, data: Array[Byte]): Unit = {
     logger.info("[frameworkMessage] executor:" + Str.id(executorId.getValue) + " slave:" + Str.id(slaveId.getValue) + " data: " + new String(data))
+
+    val broker = cluster.getBroker(Broker.idFromExecutorId(executorId.getValue))
```

Comment:
Now that we are utilizing this more we need a way for different messages to come back and be processed and forwarded to other events in the scheduler. I don't think this approach makes sense because next we wanted to-do logging and some like features it will be cluttered code and hard to support. We may need a few diagrams with the Akka parts and some notes around it on the wiki of the github (seems like a good enough place) but a few changes in that regard might make sense here (doesn't have to be Akka (not advocating the framework vs the approach (at least right now)))